pkg/cloudprovider/providers/openstack/metrics/metrics.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
```

Comment:
I think we might want to move this to a generic location so that other component can benefit it later on

Reply:
Thank you everyone for reviewing and approving this PR.
Refactoring / moving this to a generic location is handled within https://github.com/kubernetes/cloud-provider-openstack/pull/1236.